BrightFuturesTests/ResultTests.swift

```diff
@@ -151,26 +161,26 @@ class ResultTests: XCTestCase {
         XCTAssert(r.isFailure)
         XCTAssertEqual(r.error!.domain, "DivisionByZeroError")
     }
-    
-    func testRecoverNeeded() {
```

Comment:
Just to summarize: [we’d be happy to accept a PR for `recover`/`recoverWith`](https://github.com/antitypical/Result/issues/41#issuecomment-101460139) :bow: